src/atoms/Switch/Switch.native.js

```diff
@@ -0,0 +1,315 @@
+import React, { useState, useContext, useRef, useEffect } from 'react';
+import { Animated, TouchableOpacity } from 'react-native';
+import styled, { ThemeContext } from 'styled-components/native';
+import PropTypes from 'prop-types';
+
+import { makePxValue } from '../../_helpers/theme';
+import automation from '../../_helpers/automation-attributes';
+import View from '../View/';
+import isPropDefined from '../../_helpers/isPropDefined';
+
+const styles = {
+  size({ size, theme }) {
```

Comment:
be more explicit. size.container and size.knob like that don't return objects like this we don't follow that convention anywhere